code/modules/mapping/helpers/wingrille_spawner.dm

```diff
@@ -92,6 +92,7 @@
 	win_path = /obj/window/auto
 	full_win = TRUE
 	no_dirs = TRUE
+	layer = LATTICE_LAYER	// to stop it rendering over things in map editors
```

Comment:
Won't this render below turfs? Or is that not how lattices work?

Reply:
the lattice layer is actually TURF_LAYER+0.1 so that they can appear on plating, so it should be okay.